### ISSUE TITLE
[1.20.4] Game crashes during startup due to mixin inject failure

### ISSUE THREAD

**Issue by GenericCherrySwitch** [JSJBDEV/GiveMeHats#32]
Attempting to launch the game with game version 1.20.4, loader version 0.15.11 with the mod installed crashes the game at startup due to a mixin-related problem.

The error entry, along with the crash log are respectively linked below:
https://gist.github.com/GenericCherrySwitch/54172d21a764fb07f186ae51cc7022e7
https://gist.github.com/GenericCherrySwitch/a43a95e8a0dcc325bedcbcf67f98e66b

**Comment by JSJBDEV** [comment]
yeah i think they re-wrote half the code base in 1.20.4 or .5, likely I should just remove the tag I wasn't sure which version they did this on, so I assume its 1.20.4 in this case. -
